### ISSUE TITLE
bug: select should be clickable from the label

### ISSUE THREAD

**Issue by brandyscarney** [ionic-team/ionic#13244]
**Ionic version:**  (check one with "x")
(For Ionic 1.x issues, please use https://github.com/ionic-team/ionic-v1)
[ ] **2.x**
[ ] **3.x**
[x] **4.x**

**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request

Currently can't click anywhere on an item to open a select in `core`. Should be able to click anywhere and open the select like v3.

**Comment by dylanvdmerwe** [comment]
https://github.com/ionic-team/ionic/issues/10725

**Comment by brandyscarney** [comment]
@dylanvdmerwe Possibly, I haven't looked into toggle. :smile: This issue was actually created for v4 as I added select and wanted an issue for it. I've fixed it though on this PR: https://github.com/ionic-team/ionic/pull/13769

The solution for this is rather simple, I just change the position of the select when it is inside of an item so the clickable element attaches to the item:

```
.item .select {
  position: static;
}
```

Toggle is a little more complicated though with gestures.
